renv/core.py

```diff
@@ -186,12 +201,14 @@ def ensure_directories(self, env_dir):
         context.env_R_libs = r_env_libs
         context.abs_R_libs = r_abs_libs
         context.env_R_include = os.path.join(env_dir, r_env_include)
+        context.env_R_doc = os.path.join(env_dir, r_env_doc)
+        context.env_R_share = os.path.join(env_dir, r_env_share)
         context.env_bin_path = binpath = os.path.join(env_dir, binname)
         context.bin_path = binpath
         context.env_R_exe = os.path.join(binpath, r_exe)
         context.env_R_script = os.path.join(binpath, r_script)
         utils.create_directory(context.env_R_libs, self.clear)
-        logging.info(f"Environment R:  {r_env_home}")
+        logging.info("Environment R:  %s" % r_env_home)
         return context
 
     def create_configuration(self, context):
```

Comment:
This function is huge. Could we break this down into smaller functions? Same go with some of the other functions.

Reply:
See my previous comment.